src/main/scala/wdl4s/wdl/TaskOutput.scala

```diff
@@ -11,6 +13,10 @@ object TaskOutput {
     val expression = WdlExpression(ast.getAttribute("expression"))
     TaskOutput(name, wdlType, expression, ast, parent)
   }
+  
+  def buildWomOutputDefinition(taskOutput: TaskOutput) = OutputDefinition(taskOutput.unqualifiedName, taskOutput.wdlType, PlaceholderExpression(taskOutput.wdlType))
 }
 
-case class TaskOutput(unqualifiedName: String, wdlType: WdlType, requiredExpression: WdlExpression, ast: Ast, override val parent: Option[Scope]) extends Output
+case class TaskOutput(unqualifiedName: String, wdlType: WdlType, requiredExpression: WdlExpression, ast: Ast, override val parent: Option[Scope]) extends Output {
+  lazy val womOutputDefinition = TaskOutput.buildWomOutputDefinition(this)
```

Comment:
Yeah I was mostly following the existing pattern that was putting the WDL -> WOM logic into the companion object and calling it like this in the class with a reference to `thi(t)s(elf)`.
I'm happy to change it as long as it's consistent everywhere.
@cjllanwarne might also have an opinion on this